export_layers.inx

```diff
@@ -6,45 +6,48 @@
-    <param name="help" type="description">  </param>
-    <param name="help" type="description">  Output filename is of the form: </param>
-    <param name="help" type="description">    /path/to/output/directory/[subdirectory/][prefix_][number_]layer_name.ext  </param>
-    <param name="help" type="description">  with [subdirectory], [prefix] and [number] being optional </param>
-    <param name="help" type="description">  </param>
-    <param name="help" type="description">  If you pick a file below,     </param>
-    <param name="help" type="description">  - Its directory will be the '/path/to/output/directory/'.   </param>
-    <param name="help" type="description">  - Its filename will start with [prefix] (leave filename empty for no prefix)   </param>
-    <param name="help" type="description"> If you leave this box empty, default output directory will be '~/' with no prefix    </param>
-    <param name="output-source" type="path" _gui-text="Source file"></param>
-    <param name="help" type="description">     </param>
-    <param name="help" type="description"> Name of [subdirectory/] to export to (leave empty for no subdirectory)     </param>
-    <param name="output-subdir" type="string" _gui-text="Sub-directory">layers/</param>
-    <param name="help" type="description">     </param>
-    <param name="enumerate" type="boolean" _gui-text="Add layer {number} to filenames ('001', '002', etc.)">false</param>
-    <param name="help" type="description">     </param>
+    <label>
+        Export layers to different files<br/>
+        - Layers with a name starting with "[fixed]" or "[f]" are always exported.<br/>
+        - Layers with a name starting with "[export]" or "[e]" are progressively exported in their respective order.<br/>
+        - All other layers are ignored.<br/>
+        - Sublayers and groups inside exported layers are exported if visible.
+    </label>
+
+    <label/>
+
+    <label>If the output directory is not specified, it will be "~/" by default.</label>
+    <param name="output-dir" type="path" mode="folder" _gui-text="Output directory"></param>
+
+    <label/>
+
+    <label>An output filename is [prefix][number_]label.ext".</label>
+    <param name="prefix" type="string" _gui-text="Filenames prefix"></param>
+    <param name="enumerate" type="boolean" _gui-text="Also add layer number to filenames ('001', '002', etc.)">false</param>
+
+    <label/>
 
     <param name="fit-contents" type="optiongroup"  _gui-text="Area to export">
         <option value="false" selected="selected">Use document boundaries</option>
         <option value="true">Use layer content boundaries</option>
     </param>
 
-    <param name="help" type="description">     </param>
-    <param name="help" type="description"> Show all exported layers below the current layer.     </param>
+    <label/>
+
+    <label>Show all exported layers below the current layer.</param>
```

Comment:
Inkscape 1.2 complains about a lable mismatch here.

Reply:
changing the last `</param>` for <`/label>` fixes the issue

Reply:
Yep, thanks. I've found this issue too.